### PR TITLE
Add missing ALS sequence diagrams 

### PR DESCRIPTION
**Pull request by rmothilal**
Diagrams added:
- Admin:
  - GET /oracles
  - POST /oracles
  - PUT /oracles/{ID}
  - DELETE /oracles/{ID}

- API: 
  - DELETE endpoint cache

- Arch Diagrams for ALS Admin

### CODE REVIEW

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-admin-delete-oracle-7.3.4.puml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Rajiv Mothilal <rajiv.mothilal@modusbox.com>
+ --------------
+ ******'/
+
+
+@startuml
+' declare title
+title 7.3.3 Delete Oracle Endpoint
```
Comment (elnyry-sam-k) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-admin-delete-oracle-7.3.4.puml:28]: File name says 7.3.4

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-admin-post-oracle-7.3.2.puml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Rajiv Mothilal <rajiv.mothilal@modusbox.com>
+ --------------
+ ******'/
+
+
+@startuml
+' declare title
+title 7.3.1 Create Oracle Endpoints
```
Comment (elnyry-sam-k) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-admin-post-oracle-7.3.2.puml:28]: Please sync with file name

**Comment by elnyry-sam-k** [comment]
Thanks @rmothilal .. one minor question - is there really a specific handler for every type of functionality (create, update, delete, etc)?

**Comment by rmothilal** [comment]
> Thanks @rmothilal .. one minor question - is there really a specific handler for every type of functionality (create, update, delete, etc)?

@elnyry-sam-k yes each type of functionality has its own Handler function

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**